lisa/util/__init__.py

```diff
@@ -231,3 +234,11 @@ def is_valid_url(url: str, raise_error: bool = True) -> bool:
 
 def filter_ansi_escape(content: str) -> str:
     return __ansi_escape.sub("", content)
+
+
+def dump_file(file_name: Path, content: Any) -> None:
+    file_name.absolute().relative_to(constants.RUN_LOCAL_PATH)
+    file_name.parent.mkdir(parents=True, exist_ok=True)
+    file = open(file_name, "w")
+    file.write(secret.replace(content))
+    file.close()
```

Comment:
Please organize the commits with `git rebase -i ...`, `git add -p ...` . Make each commit is meaningful, independent. In this commit, one or two commits both ok.

Reply:
In the last sentence did you mean that you want this pull request to only have one or two commits?